index.html

```diff
@@ -5,9 +5,62 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/c6f4ddbd48.js"
+      crossorigin="anonymous"
+    ></script>
     <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>My portfolio</h1>
+    <main>
+      <section class="landing">
+        <nav class="nav_bar">
+          <div class="toolbar">
+            <h1 class="welcome">Welcome</h1>
+            <ul>
+              <li><a href="">Portfolio</a></li>
+              <li><a href="">Contact</a></li>
+              <li><a href="">About</a></li>
+            </ul>
+            <span class="menu_icon"><i class="fa-solid fa-bars"></i></span>
```

Comment:
- [ ] Please use a `nav` tag for your hamburger button, as it will be used for the navigation of our portfolio page and it will also come in handy when you do the desktop version.